File: clients/go/my_strategy.go
package main

import (
	"aicup22/pkg/model"
	"aicup22/pkg/types"
)

type MyStrategy struct{}

func NewMyStrategy(constants model.Constants) *MyStrategy {
	return &MyStrategy{}
}

func (strategy MyStrategy) getOrder(game model.Game, debugInterface *types.DebugInterface) model.Order {
	return model.Order{
		UnitOrders: make(map[int32]model.UnitOrder),
	}
}

func (strategy MyStrategy) debugUpdate(displayedTick int32, debugInterface types.DebugInterface) {}

func (strategy MyStrategy) finish() {}

File: clients/csharp/MyStrategy.cs
namespace AiCup22
{
    public class MyStrategy
    {
        public MyStrategy(AiCup22.Model.Constants constants) {}
        public AiCup22.Model.Order GetOrder(AiCup22.Model.Game game, DebugInterface debugInterface)
        {
            return new AiCup22.Model.Order(new System.Collections.Generic.Dictionary<int, AiCup22.Model.UnitOrder>());
        }
        public void DebugUpdate(DebugInterface debugInterface) {}
        public void Finish() {}
    }
}